### DISCUSSION TITLE
Struggling to load snippets for one filetype in another

### DISCUSSION THREAD

**Discussion by ghost** [L3MON4D3/LuaSnip#968] [Q&A] [answered]
I'm trying to load my typescript snippets in svelte files using `extend_load_ft`, but either I'm missing something or it doesn't work. Here's my config:
```lua
return {
  "L3MON4D3/LuaSnip",
  config = function()
    require("luasnip").setup {
      region_check_events = { "CursorHold", "InsertLeave", "CursorMoved" },
      delete_check_events = { "TextChanged", "InsertEnter" },
      store_selection_keys = "<C-x>",
      enable_autosnippets = true,
      load_ft_func = require("luasnip.extras.filetype_functions").extend_load_ft {
        svelte = { "typescript" },
      },
    }

    require("luasnip.loaders.from_lua").lazy_load {
      paths = "./lua/user/luasnips",
    }
  end,
  build = "make install_jsregexp",
}
```

I tried multiple different filetypes, swapping them, rotating them, but the typescript snippets don't seem to get loaded when I'm in a svelte file. What am I missing?

**Comment by L3MON4D3** [comment, marked as answer]
Do you also have an `ls.filetype_extend("svelte", {"typescript"})` somewhere?
`load_ft_func` only takes care of loading, which is (maybe a bit unexpectedly) unrelated to the actually active filetypes.

**Reply by ghost** [reply]
Yeah, I completely missed that, thanks, that solved it!
